src/pocketmine/entity/Entity.php

```diff
@@ -655,6 +655,14 @@ public function setImmobile(bool $value = true){
 		$this->setGenericFlag(self::DATA_FLAG_IMMOBILE, $value);
 	}
 
+        public function isInvisible() : bool{
+                return $this->getGenericFlag(self::DATA_FLAG_INVISIBLE);
+        }
+
+        public function setInvisible(bool $value = true){
```

Comment:
can have a void typehint here

Reply:
oh totally forgot about tabs, do I strict type other like-functions too?

Reply:
No, keep it to related changes. Others can be changed separately.